### DISCUSSION TITLE
Meteor M N2-3

### DISCUSSION THREAD

**Discussion by ysamouhos** [jekhokie/raspberry-noaa-v2#592] [General]
I am trying different settings to OQPSK etc and  I am using either rtl_fm or gnuradio but without luck. 

Anyone had any success with new Meteor M2-3 ?

**Comment by ShianMull** [comment]
I’ve been trying similar.  Meteor-M2 3 TLEs are now available in the updates downloaded, but even changing the schedule scripts does not seem to schedule M2 3 passes.  I think there is some embedded references to ‘Meteor M2’ in the various components scripts etc of RaspberryNOAA that need the authors to have a look at.

Any of the code owners able to suggest if we can just modify scripts etc ourselves or the code needs an update?

Cheers

Mark

**Comment by ysamouhos** [comment]
Here are my changes in order to schedule and capture. 

In scripts/schedule_captures.sh I changed "METEOR-M 2"  to the correct name of N2-3 in TLE like bellow. 
```

if [ "$OBJ_NAME" == "METEOR-M2 3" ]; then
  SAT_MIN_ELEV=$METEOR_M2_SAT_MIN_ELEV
fi

```
also in scripts/schedule.sh I did the following changes. 

```
if [ "$METEOR_M2_SCHEDULE" == "true" ]; then
  log "Scheduling Meteor-M2 3 captures..." "INFO"
  $NOAA_HOME/scripts/schedule_captures.sh "METEOR-M2 3" "receive_meteor.sh" $TLE_OUTPUT $start_time_ms $end_time_ms >> $NOAA_LOG 2>&1
fi
```
and.. 

```
#  grep "METEOR-M 2" $WEATHER_TXT -A 2 >> $TLE_OUTPUT
  grep "METEOR-M2 3" $WEATHER_TXT -A 2 >> $TLE_OUTPUT

```

I hope this solves your issue.

**Reply by ShianMull** [reply]
Hi, yes they were the settings I have tried with GNURADIO but no success to date.  Did you actually receive and decode images?  

I’ve just tried copying TLEs for M2 3 into the M2 section of the filtered data for relevant satellites.  Just seeing if it works with everything else set at METEOR M2 since M2 3 is now broadcasting on 137.100.  Need another couple of days before this filters through in the schedule now I’ve been mucking about with everything else.

**Reply by ysamouhos** [reply]
Are you sure it's 137.1 and not 137.9? I forgot to mention I also changed scripts/common.sh:METEOR_FREQ="137.9125" but if now that changed I will have to revert to 137.1000.

**Reply by ShianMull** [reply]
Yes, it was changed earlier this week from 137.9 to a preferred 137.1.  But, this is a Russian bird so they might change at any time.  Corroboration on change in frequency here: https://usradioguy.com/meteor-m2-3/

Cheers

Mark

**Reply by ysamouhos** [reply]
Ahhh didn't see that :) Thanx. 
I think I am going to take it step by step, 
1. Record with gnuradio and then decode in windows
2. Record with rtl_fm and again decode in windows. 
3. If any of the above goes well then I will try again meteor_demod, etc..

**Reply by ShianMull** [reply]
Just an update that even with the right TLEs giving scheduling for Meteor-M2 3, the current Meteor configuration in RaspberryNOAA is not decoding the audio (it is a different resolution etc from Meteor-M 2 so probably not surprising).  I hope you have success with the back to windows tests.  I’ll have a look for where the decode parameters are set and try to match to Meteor-M2 3, unless anyone with intimate knowledge of the code can quickly update in the meantime?

**Reply by chrisd65** [reply]
In which line numbers did you insert the two lower line blocks? They're not there, so I'm guessing you added them? In which lines?

**Reply by ysamouhos** [reply]
Not sure I understand the question. Can you please elaborate a bit?

**Reply by Bill1716** [reply]
chrisd65 - Do you mean commands starting with grep? On line 75 (approximately).

**Reply by ysamouhos** [reply]
Find this line `grep "METEOR-M 2" $WEATHER_TXT -A 2 >> $TLE_OUTPUT`
and replace it with `grep "METEOR-M2 3" $WEATHER_TXT -A 2 >> $TLE_OUTPUT`

**Comment by overclock25** [comment]
> I am trying different settings to OQPSK etc and I am using either rtl_fm or gnuradio but without luck.
> 
> Anyone had any success with new Meteor M2-3 ?

No unfortunately. Hope that somebody will publish the solution.

**Comment by ysamouhos** [comment]
Just an update here. 

Based on this `https://github.com/happysat/Meteor-M-N-2-3-Satellite-Operational-Status/blob/main/README.md`

There were a number of switches and switchbacks on Freq and SymbolRate so one has to keep track of it because it's not yet production stable. It took me a lot of failed attempts to record before I find the changes that happened in last 15 days or so. 

Now my next problem is that my QFH antenna does not give me a strong enough signal and also found (the hard way) that Nooelec NOAA+ LNA does not work with closed loop antennas so I was forced to remove the LNA (which btw has 2 filters, one on input and one on output which is very nice and very sad at the same time ) and work with only the gain of my  RTL2838UHIDIR. 

I will build a turnstile antenna and see if there is any better.
